include/envoy/registry/registry.h

```diff
@@ -261,9 +261,9 @@ template <class Base> class FactoryRegistry : public Logger::Loggable<Logger::Id
     absl::string_view canonicalName = canonicalFactoryName(name);
 
     // Next, disable the factory by all its deprecated names.
-    for (const auto& entry : deprecatedFactoryNames()) {
-      if (entry.second == canonicalName) {
-        disable(entry.first);
+    for (const auto& [name, instead_name] : deprecatedFactoryNames()) {
```

Comment:
I'd suggest `deprecated` and `canonical`; "instead_name" seems a little strange.

Reply:
will do